bff/cmd/main.go

```diff
@@ -55,9 +56,20 @@ func run(webappPath string) error {
 	var config Config
 	config.WebappPath = webappPath
 	config.Port = rawConfig["port"].(string)
+
+	// Get access key from environment variable with fallback to config file
+	accessKey := os.Getenv("ACCESS_KEY")
+	if accessKey == "" {
+		accessKey, ok := rawConfig["access_key"].(string)
+		if !ok || accessKey == "" {
+			accessKey = "DEFAULT-ACCESS-KEY"
```

Comment:
It would be more secure if we simply fail to launch if the access key is not provided. Setting the default access key should be part of the Docker compose / `entrypoint.sh` for the single server / dev setup.

Reply:
Done.